docs/1-getting-started.md

```diff
@@ -1,92 +1,82 @@
 
 ## Contents
 
-1. [Software](#software)
-2. [Email Encryption](#email-encryption)
-3. [SMTP Settings and App Passwords](#smtp-settings-and-app-passwords)
+1. [Email Encryption](#email-encryption)
+2. [Message Forwarding](#message-forwarding)
 
-<img src="img/family.cover.png">
+## Email Encryption
 
-## Prerequisites
+We strongly encourage all Hush Line operators to add a PGP key to ensure messages are only technically accessible by the key owner. 
 
-### 1. Email
+We'll use Mailvelope, an open-source, cross-browser extension that allows you to decrypt messages directly in the app.
 
-For Hush Line to email your message, you'll need SMTP information from your mail provider. We'll use Gmail for its high reliability. If you don't already have an account, create one. Maintaining an email address separate from your personal account is a good practice.
+### Step 1: Get the extension
 
-You should never use your primary account password; instead, create app-specific passwords. These are purpose-made for single applications.
+Mailvelope is available in Firefox and Chrome and we'll use Firefox as our example. First, click on the menu icon with three horizontal lines in the top right of your browser. Then select "Add-ons and themes".
 
-#### 1.1. Create an app password
+![step1](https://github.com/user-attachments/assets/b8a64098-9c89-4c64-a7bc-bda3bf6e884f)
 
-From your Gmail account, click on your avatar to access your account-level options. Select `Manage Google Account`.
+### Step 2: Search for Mailvelope
 
-<img src="img/14-profile-menu.png">
+In the search bar at the top of the screen, enter `Mailvelope` and select the correct result.
 
-Next, click on the Security tab on the left, then scroll to the bottom of the page to find App Passwords.
+![step2](https://github.com/user-attachments/assets/615b2516-18f5-4946-8fdb-7dc07272bfe7)
 
-<img src="img/15-security.png">
-<img src="img/16-app-passwords.png">
+### Step 3: Create a key
 
-Click on App Passwords, then enter a name, then click Create. A dialog will pop up with your app-specific password. Copy it to a notepad for easy access.
+#### Pin the extension
 
-<img src="img/17-create-password.png">
-<img src="img/18-copy-password.png">
+First, we'll pin the extension to our toolbar. Click on the puzzle piece icon, then the settings gear icon for Mailvelope. Select "Pin to Toolbar." When the icon appears, click it, then select "Let's start!".
 
-#### 1.2. SMTP Settings
+![step3](https://github.com/user-attachments/assets/ce1eac41-b4ad-4048-b5a2-765fbb5cb36c)
 
-##### Gmail SMTP
-- Server: `smtp.gmail.com`
-- Port: `587` or `465`
+#### Generate your key
 
-##### Riseup.net SMTP
-- Server: `mail.riseup.net`
-- Port: `587` or `465`
+In the Setup dashboard, click on the "Generate key" button. Add a name for your key, your email address, and a strong password. Before clicking "Generate," click on the "Advanced" button. In the Algorithm picklist, choose "ECC - Curve25519". Now, generate your key!
```

Comment:
If we're telling folks to do something "advanced" we should give them a reason why
```suggestion
In the Setup dashboard, click on the "Generate key" button. Add a name for your key, your email address, and a strong password. Before clicking "Generate," click on the "Advanced" button. In the Algorithm picklist, choose "ECC - Curve25519". This algorithm is more efficient that traditional RSA encryption while providing a similar level of security. Now, generate your key!
```